roles/post-db-restore/tasks/main.yml

```diff
@@ -62,11 +62,13 @@
   with_items:
     - "-y vset install_profile stanford_dept"
     - "cc all"
-    - "sqlq 'update system set status=\"1\" where name=\"stanford_dept\"'"
+#    - "sqlq 'update system set status=\"1\" where name=\"stanford_dept\"'"
```

Comment:
@sherakama, yeah the problem is that `drush sqlq 'update system set status="1" where name="stanford_dept"'` doesn't work.

Bill's approach _does_ resolve the issue with running `drush rr` later. I'm just not sure about the ramifications of re-running the install profile. I'm trying to figure out what `drush en` is doing that the SQL query is not.

Reply:
Running the installation profile after the database restore sounds like a bad idea as the profiles execute a lot of configuration code and could enable dependencies that you do not want. I would advise against running the install.

Reply:
https://api.drupal.org/api/drupal/includes%21module.inc/function/module_enable/7.x

I would start here

Reply:
I wonder if registry rebuild disables the profile because some dependencies aren't enabled.